.github/workflows/build_pip.yaml

```
@@ -52,9 +53,9 @@ jobs:
 
       - name: Build conda package
         run: |
-          pip install --no-cache-dir cython pytest hypothesis
+          pip install --no-cache-dir cython
           pip install --no-cache-dir numpy ${{ matrix.use_pre }}
           echo "CONDA_PREFFIX is '${CONDA_PREFIX}'"
           export MKLROOT=${CONDA_PREFIX}
-          pip install -e . --no-build-isolation --verbose --no-deps
+          pip install -e .[test] --no-build-isolation --verbose
           python -m pytest -v mkl_fft/tests
```

t might be helpful to add `pip list` here to dump the test env:
```suggestion
          pip list
          python -m pytest -v mkl_fft/tests
```